### ISSUE TITLE
does not work on Lunar Client 

### ISSUE THREAD

**Issue by tnhrasv123** [DFOnline/CodeClient#90]
**Describe the bug**
the mod does not work on Lunar Client.

**To Reproduce**
1. Add the mod and yet another config lib to the mod folder on lunar client 1.20.4.
2. try to open the game.

**Expected behavior**
Minecraft opens for a brief period and then crashed

**Checks**
Please check all these for more infomation with the bug.
- [v] I am using the [latest version](https://github.com/DFOnline/CodeClient/releases).
- [v] I have tried disabling various options to see if they cause the issue, such as new ones or ones I have recently enabled.

**Comment by tnhrasv123** [comment]
any answers?

**Comment by GeorgeRNG** [comment]
please provide a crash log (i think lunar has weird obfuscation)

lunar is not priority, i believe it has been used to break diamondfire rules and broken features like clickable chat.
it could potentially get banned by the server, like minimap mods, so you could instead use a performance modpack

**Comment by tnhrasv123** [comment]
thanks

**Comment by sprydagger** [comment]
I don't believe the mod works for 1.20.4, unless I am mistaken. The latest version of the mod is for 1.20.3?
